python/ray/dashboard/metrics_exporter/actions.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import typing
+
+from ray.dashboard.metrics_exporter.schema import ActionType, KillAction
+
+logger = logging.getLogger(__name__)
+
+
+class ActionHandler:
```

Comment:
I can move it to `client.py` if you think this file is too tiny to be here.

Reply:
this is fine, since i expect this file to grow with log action